apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -2386,72 +2442,155 @@ mod test_finalize_block {
         let root_post = shell.shell.wl_storage.storage.block.tree.root();
         assert_eq!(root_pre.0, root_post.0);
 
-        // Check first inner tx hash has been removed from storage but
-        // corresponding wrapper hash is still there Check second inner
-        // tx is still there and corresponding wrapper hash has been removed
-        // since useless
-        assert_eq!(event[0].event_type.to_string(), String::from("accepted"));
+        // Check inner tx hash has been removed from storage
+        assert_eq!(event[0].event_type.to_string(), String::from("applied"));
         let code = event[0]
             .attributes
             .get("code")
-            .expect("Test failed")
+            .expect("Testfailed")
```

Comment:
tiny nit, for cases like this where there's no meaningful message you can just use `unwrap`